circleci-orb/scripts/rebase-on-target-branch/index.ts

```diff
@@ -1,15 +1,16 @@
-#! /usr/bin/env node
+#!/usr/bin/env node
 
-// tslint:disable:no-console
 import {execSync as execSync_} from 'child_process';
 
 // CircleCI environmental variables.
 const baseRevision = process.env.CIRCLE_GIT_BASE_REVISION!;
```

Comment:
These variables are not defined though. We custom-defined them in our `config.yml`. We need to keep the Yaml input pass-through I think.

Reply:
Added those two back in place.